cmd/temporalite/main.go

```diff
@@ -133,6 +140,12 @@ func buildCLI() *cli.App {
 					return cli.Exit(fmt.Sprintf("bad value %q passed for flag %q", c.String(logFormatFlag), logFormatFlag), 1)
 				}
 
+				switch c.String(logLevelFlag) {
+				case "debug", "info", "warn", "error", "fatal":
+				default:
+					return cli.Exit(fmt.Sprintf("bad value %q passed for flag %q", c.String(logLevelFlag), logLevelFlag), 1)
```

Comment:
Thanks for the validation here!

Small nit -- it would be great if the error message included the set of allowed values, with the same source of truth used to generate the `log-level` flag's usage string. However this isn't high priority since the allowed values should be pretty static, so I'll leave it open to further contribution.